ci_publish.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -e # Exit with nonzero exit code if anything fails
 
+openssl aes-256-cbc -K $encrypted_c2f58914b0fd_key -iv $encrypted_c2f58914b0fd_iv \
+	-in secret-keys.gpg.enc -out "${SIGNING_KEYRING_FILE}" -d
```

Comment:
This approach was tested with https://github.com/ePages-de/restdocs-wiremock/pull/75#issuecomment-797508933